lib/iris/config.py

```diff
@@ -66,11 +66,25 @@
 
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
-
+import six
 from six.moves import configparser
+
+import contextlib
+from multiprocessing import cpu_count
+from multiprocessing.pool import ThreadPool
+import re
 import os.path
 import warnings
 
+import dask
+import dask.multiprocessing
+
+DISTRIBUTED_AVAILABLE = True
+try:
+    import distributed
+except ImportError:
+    DISTRIBUTED_AVAILABLE = False
```

Comment:
@dkillick Would you consider this slightly neater pattern (IMHO):
```python
try:
    import distributed
except ImportError:
    distributed = None
```
Then later in the code, rather than checking for 
```python
if DISTRIBUTED_AVAILABLE:
```
it's simply
```python
if distributed is not None:
```
See [here](https://github.com/SciTools/iris/pull/2462/files#diff-36cf2602a7c220f674c997618f233af7R342) ...